src/_connection.txt

```diff
@@ -1,3 +0,0 @@
-RabbitMQ
```

Comment:
I think this should stay. It's been in the repository for some time to make the local setup more convenient. https://github.com/Particular/ServiceControl/blob/master/docs/running-acceptance-tests.md#connection-file